pyproject.toml

```diff
@@ -421,7 +421,7 @@ bad-functions = ["map", "input"]
 # ignored-parents =
 
 # Maximum number of arguments for function / method.
-max-args = 9
+max-args = 10
```

Comment:
if method has more than 9 arguments, it's a substandard code. refactor the methods when this rule triggers. revert the change to pylint configuration.

Reply:
Done